homeassistant/components/comelit/strings.json

```diff
@@ -31,5 +31,24 @@
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "zone_status": {
+        "state": {
+          "alarm": "Alarm",
+          "armed": "Armed",
+          "open": "Open",
+          "excluded": "Excluded",
+          "faulty": "Faulty",
+          "inhibited": "Inhibited",
+          "isolated": "Isolated",
+          "rest": "Rest",
+          "sabotated": "Sabotated",
+          "unavailable": "Unavailable",
```

Comment:
If a zone is unavailable, should we set the whole entity to just _Unavailable_?

Reply:
Good point, done